codegen/src/main/scala/akka/grpc/gen/javadsl/Service.scala

```diff
@@ -59,6 +55,22 @@ object Service {
   private[javadsl] def basename(name: String): String =
     name.replaceAll("^.*/", "").replaceAll("\\.[^\\.]*$", "")
 
+  private[javadsl] def outerClass(t: FileDescriptor) =
```

Comment:
Could be regular `private` so that Java users don't see it? Otherwise, `@InternalApi` might be good.

Reply:
It's used from `Method` as well so it can't be regular `private`.

Good point, though in this case this code is *generating* the javadsl, so I don't think it'll ever be used from Java.